### PR TITLE
docs: Add initial landing page README.

### PR DESCRIPTION
**Pull request by jimouris**
Add Nillion image along with a description and a couple of links.

**Note: It's pretty simple right now, open to ideas!**

### CODE REVIEW

**Review by lumasepa — APPROVED** [review]

**Comment by navasvarela** [comment]
doc: ?

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Our goal is to build humanity's first blind computer. Nillion is a secure
+computation network that decentralizes trust for high value data in the same way
+that blockchains decentralized transactions.
+
+Get started with Nillion in one of the following links:
+* <a href="https://nillion.com">Website</a>
+* <a href="https://docs.nillion.com">Documentation</a>
+* <a href="https://twitter.com/nillionnetwork">X (Twitter)</a>
+* <a href="https://warpcast.com/nillion">Warpcast</a>
+* <a href="https://www.linkedin.com/company/nillionnetwork/mycompany">LinkedIn</a>
```
Comment (navasvarela) [inline profile/README.md:22]: Shall we add discord and telegram links?

Comment (jimouris) [reply profile/README.md:22]: done

**Review by cmacrae — APPROVED** [review]

**Review by navasvarela — APPROVED** [review]
Nice! Thanks!

**Comment by jimouris** [comment]
@cmacrae Are we using `bors` here or should I just merge?

**Comment by navasvarela** [comment]
> @cmacrae Are we using `bors` here or should I just merge?

Just merge, there's no bors in this repo

**Pull request merged**